### ISSUE TITLE
Single quote and coloring

### ISSUE THREAD

**Issue by jief666** [de-jcup/eclipse-bash-editor#40]
Syntax coloring in blue starts at every single quote, even if it's a \' (in my case an apostrophe in echo command).

**Comment by de-jcup** [comment]
@jief666 Please give me example code - and if possible a screenshot too.

**Comment by de-jcup** [comment]
@jief666 
I think I found a problemetic code by myself:

```
echo This shouldn\'t be a problem
echo 'Or did you expect this to be?'

```

this results in 
![grafik](https://user-images.githubusercontent.com/11644753/30342024-95c61a06-97f8-11e7-804c-25a8bc679ba9.png)
which is not correct. Did you mean this bug?

**Comment by de-jcup** [comment]
After change to single line variant I still got following problem:
![grafik](https://user-images.githubusercontent.com/11644753/30345687-620430c8-9806-11e7-831f-3953c3fb6985.png)

So it seems to be necessary to create an own rule which works/does syntax highlighting only, when more then one apostrophe is used

**Comment by de-jcup** [comment]
After implementing own bash string rule the output is shown as in next picture:
#![grafik](https://user-images.githubusercontent.com/11644753/30348229-f1f5d882-980e-11e7-82d5-d91f5005eaf2.png)

There are still some tests necessary to check side effects.

**Comment by de-jcup** [comment]
Strings can be mutli lined in bash, so the former reduction to a single line was not correct.

- [ ] The new BashStringLineRule has to be enabled as multi line again

**Comment by jief666** [comment]
Yes, that was this bug.
I think the solution is : if it's \', just treat as a normal char, like bash do.
I didn't understand what you meant by "bash rule"

**Comment by de-jcup** [comment]
@jief666 sorry about the "bash rule", I didn't want to confuse. It's just a name for an implementation part inside my code. I used the issue to document my changes - maybe a little bit too much.

**Comment by de-jcup** [comment]
Fix done - will come with Release 0.9.2. See screen shot for fixed colouring (dark theme variant):
![image](https://user-images.githubusercontent.com/11644753/30500965-e4fe5452-9a60-11e7-98a7-e2083425504d.png)
